apis/generate.go

```diff
@@ -3,6 +3,22 @@
 // See the below link for details on what is happening here.
 // https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module
 
+// NOTE(@negz): See the below link for details on what is happening here.
```

Comment:
Remove this line?
It's duplicate to above line.

Reply:
sure, thanks